src/main/java/com/coreos/jetcd/EtcdWatch.java

```diff
@@ -0,0 +1,32 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.options.WatchOption;
+import com.coreos.jetcd.util.ListenableSetFuture;
+import com.coreos.jetcd.watch.Watcher;
+import com.google.protobuf.ByteString;
```

Comment:
@oza thanks for your great suggestion. If we expose java String to users, and convert it to ByteString, wdyt?